packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js

```diff
@@ -87,7 +90,7 @@ function markdownToHtml(
       });
     }
 
-    paraList += `<p>${para}</p>`;
+    paraList += textOnly ? para : `<p>${para}</p>`;
```

Comment:
@annawen1 `textOnly`, I'm wondering, if this is the right name for this. This is pretty much removing the paragraph functionality, right? Maybe we can call it `createParagraphs` and have it be default as true?

Reply:
ok!